<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConceptionPlanche</class>
 <widget class="QFrame" name="ConceptionPlanche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conception de la planche finale</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="gauche"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="droite">
     <item>
      <layout class="QHBoxLayout" name="taille">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Taille du visage : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="tailleDuVisage">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hauteur">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Hauteur du visage : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hauteurDuVisage">
         <property name="minimum">
          <number>-10</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Largeur du visage : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="largeurDuVisage">
         <property name="minimum">
          <number>-10</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="boutonSauvegarder">
       <property name="text">
        <string>Sauvegarder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
